website/docs/r/lightsail_database.html.markdown

```diff
@@ -0,0 +1,115 @@
+---
+subcategory: "Lightsail"
+layout: "aws"
+page_title: "AWS: aws_lightsail_database"
+description: |-
+  Provides a Lightsail Database
+---
+
+# Resource: aws_lightsail_database
+
+Provides a Lightsail Database. Amazon Lightsail is a service to provide easy virtual private servers
+with custom software already setup. See [What is Amazon Lightsail?](https://lightsail.aws.amazon.com/ls/docs/getting-started/article/what-is-amazon-lightsail)
+for more information.
+
+~> **Note:** Lightsail is currently only supported in a limited number of AWS Regions, please see ["Regions and Availability Zones in Amazon Lightsail"](https://lightsail.aws.amazon.com/ls/docs/overview/article/understanding-regions-and-availability-zones-in-amazon-lightsail) for more details
```

Comment:
The [URL you reference](https://lightsail.aws.amazon.com/ls/docs/overview/article/understanding-regions-and-availability-zones-in-amazon-lightsai) was last updated in May 2021.

I would instead recommend the AWS Regional Services List that is sourced from an internal AWS tool.

https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/

Reply:
Updated the link